libs/wal_decoder/Cargo.toml

```diff
@@ -20,7 +20,23 @@ tokio = { workspace = true, features = ["io-util"] }
 tonic.workspace = true
 tracing.workspace = true
 utils.workspace = true
+smallvec.workspace = true
```

Comment:
nit: I don't think we ended up actually using this?

Reply:
Good spot! https://github.com/neondatabase/neon/pull/10201/commits/0b0acef3531d218df762c39ffd6be31bcc795705